composer.json

```diff
@@ -23,6 +28,11 @@
         }
     },
     "scripts": {
+        "check": "composer run check:phpstan && composer run check:phpcs",
```

Comment:
Can be done without cli command:
`"check" : ["@check:phpstan", "@check:phpcs"]`